static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -336,6 +337,10 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
           `Adding Session Replay to your [platform] project is simple. Make sure you've got these basics down.`,
           {platform: currentPlatform?.name || currentProject.slug}
         )}
+        <div>{showJsFrameworkInstructions ? tct(
+          `Also, ensure that you have set up trace propagation in your backend projects. To learn more, [link:read the docs].`,
+          {link: <ExternalLink href="https://docs.sentry.io/product/session-replay/getting-started/#:~:text=Make%20sure%20you%27ve%20set%20up%20trace%20propagation%20in%20your%20backend%20projects." />}
+        ) : ''}</div>
```

Comment:
Is the empty div necessary (in the case where `showJsFrameworkInstructions == false`)

Reply:
nope I can remove that